servers/curated-corpus-api/prisma/schema.prisma

```diff
@@ -133,26 +133,28 @@ model Section {
 }
 
 model SectionItem {
-  id                Int             @id @default(autoincrement())
-  externalId        String          @unique @default(uuid()) @db.VarChar(255)
-  sectionId         Int
-  approvedItemId    Int
+  id                    Int             @id @default(autoincrement())
+  externalId            String          @unique @default(uuid()) @db.VarChar(255)
+  sectionId             Int
+  approvedItemId        Int
   // the initial rank of the SectionItem in relation to its siblings. used as a
   // fallback when there is no engagement/click data available. may only apply to
   // ML-generated SectionItems.
-  rank              Int?
+  rank                  Int?
   // we will not delete SectionItem history, instead will deactivate items
-  active            Boolean         @default(true)
+  active                Boolean         @default(true)
+  // client sends an array of string reasons, stored as JSON as mysql prisma does not support String[]
+  deactivateReasons    Json?
```

Comment:
just curious - have you played with this datatype locally? is it easy enough to retrieve and parse and use? do we need to do any encoding before inserting?

Reply:
we should be able to easily do something like:

```
await prisma.user.create({
  data: {
    name: "Bob",
    deactivateReasons: ["Dated", "Paywall", "Other"],
  },
})
```